core/sup_tree_core/executor_pool/memcache_manager.ex

```diff
@@ -0,0 +1,103 @@
+# Copyright(c) 2015-2018 ACCESS CO., LTD. All rights reserved.
+
+use Croma
+
+defmodule AntikytheraCore.ExecutorPool.MemcacheManager do
```

Comment:
This module (and the corresponding `GenServer`s) is used only when an object is written as a cache; it's not used for reads.
Should we rename this to `MemcacheWriter` in this regard?

Reply:
Renamed in 88bba1184d0afebfc609c901ffafd5b2f37454ae